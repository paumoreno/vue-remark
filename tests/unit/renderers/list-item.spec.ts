import VueRemarkListItem from "@/renderers/list-item.vue";
import { mount } from "@vue/test-utils";

describe(VueRemarkListItem.name, () => {
  test.each([
    [
      "when checkboxes",
      {
        propsData: {
          checked: true
        }
      }
    ],
    [
      "when regular list items",
      {
        propsData: {
          checked: null
        }
      }
    ]
  ])("%s", (_, options) => {
    const wrapper = mount(VueRemarkListItem, options);

    expect(wrapper).toMatchSnapshot();
  });
});
