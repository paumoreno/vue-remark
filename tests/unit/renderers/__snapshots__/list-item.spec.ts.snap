// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkListItem when checkboxes 1`] = `<li><input type="checkbox" readonly="readonly"> </li>`;

exports[`VueRemarkListItem when regular list items 1`] = `
<li>
  <!---->
</li>
`;
